tensorflow/lite/kernels/resize_bilinear_test.cc

```diff
@@ -465,5 +465,31 @@ TEST_P(ResizeBilinearOpTest, HorizontalResizeExtremeNegativeValuesInt16) {
 INSTANTIATE_TEST_SUITE_P(ResizeBilinearOpTest, ResizeBilinearOpTest,
                          testing::Values(TestType::kConst, TestType::kDynamic));
 
+class ResizeBilinearOpQuantizationTest
+    : public ::testing::TestWithParam<TensorType> {};
+
+TEST_P(ResizeBilinearOpQuantizationTest, MismatchedQuantizationFails) {
+  TensorType tensor_type = GetParam();
+  ResizeBilinearOpModel m({tensor_type, {1, 2, 2, 1}, 0.0f, 0.0f, 0.5f, 1},
+                          {3, 3}, TestType::kConst);
+  TfLiteTensor* output_tensor = m.GetOutputTensor(0);
+  output_tensor->params.scale = 0.25f;
+  output_tensor->params.zero_point = 2;
+  
+  if (tensor_type == TensorType_INT8) {
+    m.SetInput<int8_t>({1, 2, 3, 4});
+  } else if (tensor_type == TensorType_UINT8) {
+    m.SetInput<uint8_t>({1, 2, 3, 4});
+  } else if (tensor_type == TensorType_INT16) {
+    m.SetInput<int16_t>({1, 2, 3, 4});
+  }
```

Comment:
I _think_ you can also just set the type as the widest one and rely on type coercions and binary representations to do the right thing for the other cases. Then it will be the same as the constructor of the model, on line 473, you just list the numbers.